viewing_party/playground.py

```diff
@@ -0,0 +1,45 @@
+from main import get_new_rec_by_genre
```

Comment:
Just a note that you don't need to commit this file as we don't need to give feedback to you on it.

Reply:
okay! i was practicing the git commands and I was unsure about how to rescind the staged changes. I didn't make any meaningful changes, but I did delete comments. Is it better to just leave minor stuff until there is a major/consequential change and then commit everything?